sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormTrainingClient.cs

```diff
@@ -178,6 +187,98 @@ public virtual async Task<TrainingOperation> StartTrainingAsync(Uri trainingFile
+        /// <param name="cancellationToken">A <see cref="CancellationToken"/> controlling the request lifetime.</param>
+        /// <returns>
+        /// <para>A <see cref="CreateComposedModelOperation"/> to wait on this long-running operation. Its Value upon successful
+        /// completion will contain meta-data about the trained model.</para>
+        /// <para>Even if operation fails, a model is created in the Form Recognizer account with an "invalid" status.
+        /// A <see cref="RequestFailedException"/> will be raised containing the modelId to access this invalid model.</para>
+        /// </returns>
+        public virtual CreateComposedModelOperation StartCreateComposedModel(IEnumerable<string> modelIds, CreateComposedModelOptions createComposedModelOptions = default, CancellationToken cancellationToken = default)
+        {
+            Argument.AssertNotNull(modelIds, nameof(modelIds));
+            createComposedModelOptions ??= new CreateComposedModelOptions();
+
+            using DiagnosticScope scope = Diagnostics.CreateScope($"{nameof(FormTrainingClient)}.{nameof(StartCreateComposedModel)}");
+            scope.Start();
+
+            try
+            {
+                var modelIdsGuid = new List<Guid>();
+                foreach (var modelId in modelIds)
+                {
+                    modelIdsGuid.Add(ClientCommon.ValidateModelId(modelId, nameof(modelId)));
+                }
+
+                var composeRequest = new ComposeRequest(modelIdsGuid);
+                if (createComposedModelOptions.DisplayName.Length > 0)
+                {
+                    composeRequest.ModelName = createComposedModelOptions.DisplayName;
+                }
+
+                ResponseWithHeaders<FormRecognizerComposeCustomModelsAsyncHeaders> response = ServiceClient.ComposeCustomModelsAsync(composeRequest);
+                return new CreateComposedModelOperation(response.Headers.Location, ServiceClient, Diagnostics);
+            }
+            catch (Exception e)
+            {
+                scope.Failed(e);
+                throw;
+            }
+        }
+
+        /// <summary>
+        /// Creates a composed model from a collection of existing trained models with labels.
+        /// </summary>
+        /// <param name="modelIds">List of model ids to use in the composed model.</param>
+        /// <param name="createComposedModelOptions">A set of options available for configuring the create composed model request.</param>
+        /// <param name="cancellationToken">A <see cref="CancellationToken"/> controlling the request lifetime.</param>
+        /// <returns>
+        /// <para>A <see cref="CreateComposedModelOperation"/> to wait on this long-running operation. Its Value upon successful
+        /// completion will contain meta-data about the trained model.</para>
+        /// <para>Even if operation fails, a model is created in the Form Recognizer account with an "invalid" status.
```

Comment:
Is this true for composed model? If there is an error, service will still create the composed model?

Reply:
You are right. I deleted this from the docstrings.